CognizantStudio/aps-pdf-linking
# Text based pdf linking

What's this PR do?
This PR is to remove the JSON creation logic and use the JSON from a file. Also, to filter out the blacklist data on click of "Filter" button.

What are the relevant tickets?
https://jira.cognizantaccelerator.com/browse/LP01TC-75

Where should the reviewer start?
containers/OcrPage.js, components/OcrDataPage.js

How should this be manually tested?
1 Start the app and hit the url : http://localhost:3000
2.Click on the filter button and verify whether the data in blacklistData.js should not be listed.

Who did you pair with on this change?
Myslef

New dependencies? What are they used for?
None

Screenshots (if appropriate).

What gif best describes how this PR makes you feel?
![giphy 4](https://user-images.githubusercontent.com/25795038/47289358-feaad300-d617-11e8-9869-9f78070a65ae.gif)
